### ISSUE TITLE
Request - Moding & F14b

### ISSUE THREAD

**Issue by Shirooooo** [shdwp/dcs_liberation#73]
Hey,

first of all thank you for your hard work and keeping the engine alive. Our squadron loves it, we are playing almost every evening a mission! 

Unfortunately I am too stupid to get the application working from source code to mod it. I always get the error: "name 'Terrain' is not defined" in the operation.py. Is there a way you could do a more detailed instruction than in the development guide described on how to launch it from source code? If you have no time for that, never mind it. 

The F14 pilots are asking, if you can give us the option to store the heading alignement for the F14 or set it to stored heading alignment by default? We love cold starts and that would speed up the process for us. At the moment we are editing every missionfile before we start, but if it is not so hard to implement that would be great to have it.

**Comment by Khopa** [comment]
For the Terrain error, it probably mean that you miss the pydcs library in the Python path.

If you have installed python 3, you can run 'pip install dcs' to install it globally, which is the easiest way to set it up.

***

For the F14, on the qt_ui branch I implemented your suggestion to have the F14-B INS alignement pre-stored by default. See commit d8d17e5. 

Ideally in the future, we should provide a way to edit these kind of settings from dcs liberation directly.

If you intend to run from source, you might want to have a look at this branch (qt_ui), but it is quite different from the last release as a lot of things have been and are being reworked.

**Comment by Shirooooo** [comment]
Hey Khopa,

thank you very much for your reply and the implementation of the F14 pre-stored alignment.

I thought I have the pydcs library. 

Haha.. I wrote a very long post on what I was exactly doing and with the writing I found the problem, The problem was I did not have the "dcs" folder from pydcs in the directory of the qt_ui branch folder. Oh man I am so happy now. Also I had to insert the ui folder from the main branch. Anyways, thank you!

**Comment by Khopa** [comment]
> Also I had to insert the ui folder from the main branch. 

Don't ! :grin: 
I honestly don't know what will happen, if you try to run the old ui with the changes on this branch :sweat_smile: 

----------

In fact, if you're on the qt_ui branch, you have to run the *./qt_ui/main.py* instead of the __init__.py, it is the new UI i've been working on. Code on this branch relies on this instead of the old ui.

You will also need the very last release of pycds. You can download the last version of the dcs folder from the repo https://github.com/pydcs/dcs 

Sorry, because this is not written anywhere, i'll have to update the dev guide for this branch, because for now it is not possible to know that.

Also, i believe @wrycu is still unable to run my changes, probably because of difference between  Steam & Standalone installation file path.

----------

Also, I believe @wrycu is trying to build a new release with the old UI for maintenance, in order to make the F-16 and some fixes available a bit sooner, and he has pushed these fixes on the master branch yesterday. 

-----------

**tl;dr**
So instead of using the qt_ui branch like i suggested yesterday, you should maybe for now use the master branch now that @wrycu has started importing fix there.

**Comment by Shirooooo** [comment]
oh okay, I'm on the steam version and I could launch the qt_ui branch now. Maybe the information is helping you guys, but I do not know which way it is not working on your side. Well, I will give the master branch also a try.
